sqlglot/dialects/databricks.py

```diff
@@ -29,6 +29,16 @@ class Generator(Spark.Generator):
 
         PARAMETER_TOKEN = "$"
 
+        def cast_sql(self, expression: exp.Cast) -> str:
```

Comment:
do only databricks and teradata have this syntax? should it be used more generally?

Reply:
you don't need to check the type of expression, it will be cast

Reply:
the function? perhaps make it a class property or function that to import

there are examples of functions in dialects.py

Reply:
can't you make as hared function?